tests/src/Kernel/fixtures/markup_rendering_patterns/date_block.yml

```diff
@@ -6,10 +6,13 @@ date_block_default:
       date: '2021-05-26'
   assertions:
     count:
-      'time.bcl-date-block.d-flex.flex-column.align-items-center.justify-content-center.bg-date.text-light.pt-2.rounded-top.mw-date': 1
-      'time[datetime="2021-05-26"]': 1
-      'span.py-4.bg-white.border.border-bottom-0.w-100.fs-5.fw-medium.text-dark.text-center.text-uppercase': 1
-      'span.bg-lighter.border.w-100.text-center.fs-5.text-dark.rounded-bottom.mb-n1': 1
+      # Exclude unexpected additional <time> elements.
+      'time': 1
+      'time.bcl-date-block[datetime="2021-05-26"]': 1
+      # Exclude unexpected nested elements, no matter the tag name.
+      'time > *': 1
+      'time > * > *': 2
+      'time > * > *:nth-child(1) > *': 1
```

Comment:
I don't see the value for this, the most important assertions to run here are checking that the day and month are in the first box and year is in the second. Let's drop these altogether.

Reply:
Ok for me to remove.

However it makes me wonder, when do we ever want to have these count checks?
Obviously their purpose is to fail if we have unexpected element.

I could see two different reasons why we would not want to do this:
- We allow BCL to insert additional elements without having to update our tests.
- We trust that unexpected additional elements are unlikely to appear, so it is not worth the additional lines of testing code.

Reply:
Let's keep the `'time.bcl-date-block[datetime="2021-05-26"]': 1` for the datetime attribute.